dockerfiles/custom_images.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
```

Comment:
I don;t know how build.include works I need something right now because devs blocked to update acceptances. maybe in future we will merge them